.github/workflows/npm-publish.yml

```diff
@@ -15,9 +18,22 @@ jobs:
           node-version: "18.16.1"
       - run: npm ci
 
-  publish-npm:
+  version-bump:
     needs: build
     runs-on: ubuntu-latest
+    steps:
```

Comment:
I like this solution for version bump, looks ugly but easy to follow. No need for dependency, just the linux environment.

OK!